website/docs/r/sentinel_alert_rule_anomaly_built_in.html.markdown

```diff
@@ -0,0 +1,172 @@
+  Manages a Built-in Anomaly Alert Rule.
+---
+## Disclaimers
+
+~> **Note:** A Built-in Anomaly Alert Rule could not be deleted. delete a Terraform managed Built-in Anomaly Alert Rule will cause the Built-in Anomaly Alert Rule to be disabled.
+
+# azurerm_sentinel_alert_rule_anomaly_built_in
+
+Manages a Built-in Anomaly Alert Rule.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_log_analytics_workspace" "example" {
+  name                = "example-law"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku                 = "PerGB2018"
+}
+
+resource "azurerm_security_insights_sentinel_onboarding" "example" {
+  resource_group_name          = azurerm_resource_group.example.name
+  workspace_name               = azurerm_log_analytics_workspace.example.name
+  customer_managed_key_enabled = false
+}
+
+data "azurerm_sentinel_alert_rule_anomaly" "example" {
+  log_analytics_workspace_id = azurerm_log_analytics_workspace.example.id
+  display_name               = "UEBA Anomalous Sign In"
+  depends_on                 = [azurerm_sentinel_log_analytics_workspace_onboarding.example]
+}
+
+resource "azurerm_sentinel_alert_rule_anomaly_built_in" "example" {
+  display_name               = "UEBA Anomalous Sign In"
+  log_analytics_workspace_id = azurerm_log_analytics_workspace.example.id
+  mode                       = "Production"
+  enabled                    = false
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Optional) The Name of the built-in Anomaly Alert Rule. Changing this forces a new Built-in Anomaly Alert Rule to be created.
+
+* `display_name` - (Optional) The Display Name of the built-in Anomaly Alert Rule. Changing this forces a new Built-in Anomaly Alert Rule to be created.
+
+~> **Note:** One of `name` or `display_name` block must be specified.
+
+* `log_analytics_workspace_id` - (Required) The ID of the Log Analytics Workspace. Changing this forces a new Built-in Anomaly Alert Rule to be created.
+
+* `enabled` - (Required) Should the Built-in Anomaly Alert Rule be enabled?
+
+* `mode` - (Required) mode of the Built-in Anomaly Alert Rule. Possible Values are `Production` and `Flighting`.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported: 
+
+* `id` - The ID of the Built-in Anomaly Alert Rule.
+
+* `anomaly_settings_version` - The anomaly settings version of the Anomaly security ml analytics settings that dictates whether job version gets updated or not.
+
+* `anomaly_version` - The anomaly version of the Anomaly Alert Rule.
+
+* `description` - The description of the Anomaly Alert Rule.
+
+* `frequency` - The frequency the Anomaly Alert Rule will be run.
+
+* `is_default_settings` - Whether this Anomaly Alert Rule is a default settings.
```

Comment:
the grammar here is wrong? is this a default rule? what settings? settings should this be setting? what does this do?

should itjust be
```suggestion
* `is_default` - Whether this Anomaly Alert Rule is a default.
```

Reply:
I think it's not used, so maybe we can remove it... in this `built_in` one, it's always `true`. While in `duplicate` one, there will be some more configurable parameters, if the parameters equals to default ones, then this will be `true`.

Reply:
then sounds like we should remove it then. everything else looks good now